### PR TITLE
Add EndpointSlice blog post in pt language

### PR DESCRIPTION
**Pull request by rikatz**
Add EndpointSlice blog post in pt language

Partial fix for #13939

### CODE REVIEW

File: content/pt/blog/_posts/2020-09-02-scaling-kubernetes-networking-endpointslices.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Por padrão, o EndpointSlices armazena um máximo de 100 _endpoints_ cada, podendo isso ser configurado com a flag `--max-endpoints-per-slice` no kube-controller-manager.
+
+## EndpointSlices provê uma melhoria de escalabilidade em 10x
+Essa API melhora dramaticamente a escalabilidade da rede. Agora quando um Pod é adicionado ou removido, apenas 1 pequeno EndpointSlice necessita ser atualizado. Essa diferença começa a ser notada quando centenas ou milhares de Pods compõem um único _Service_.
+
+Mais significativo, agora que todos os IPs de Pods para um _Service_ não precisam ser armazenados em um único recurso, nós não precisamos nos preocupar com o limite de tamanho para objetos armazendos no etcd. EndpointSlices já foram utilizados para escalar um serviço além de 100,000 endpoints de rede.
+
+Tudo isso é possível com uma melhoria significativa de performance feita no kube-proxy. Quando o EndpointSlices é usado em grande escala, muito menos data será transferida para as atualizações de endpoints e o kube-proxy torna-se mais rápido para atualizar regras do iptables ou do ipvs. Além disso, os _Services_ podem escalar agora para pelo menos 10x mais além dos limites anteriores.
```
Comment (jhonmike) [inline content/pt/blog/_posts/2020-09-02-scaling-kubernetes-networking-endpointslices.md:35]:
```suggestion
Tudo isso é possível com uma melhoria significativa de performance feita no kube-proxy. Quando o EndpointSlices é usado em grande escala, muito menos dados serão transferidos para as atualizações de endpoints e o kube-proxy torna-se mais rápido para atualizar regras do iptables ou do ipvs. Além disso, os _Services_ podem escalar agora para pelo menos 10x mais além dos limites anteriores.
```

File: content/pt/blog/_posts/2020-09-02-scaling-kubernetes-networking-endpointslices.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Serviços Dual-stack são uma nova funcionalidade que foi desenvolvida juntamente com o EndpointSlices. Eles irão utilizar simultâneamente endereços IPv4 e IPv6 para serviços, e dependem do campo addressType do Endpointslices para conter esses novos tipos de endereço por família de IP.
+
+O roteamento baseado por topologia irá atualizar o kube-proxy para dar preferência no roteamento de requisições para a mesma região ou zona, utilizando-se de campos de topologia armazenados em cada endpoint dentro de um EndpointSlice. Como uma melhoria futura disso, estamos explorando o potencial de subconjuntos de endpoint. Isso irá permitir o kube-proxy apenas observar um subconjunto de EndpointSlices. Por exemplo, isso pode ser compinado com o roteamento baseado em topologia e assim, o kube-proxy precisará observar apenas EndpointSlices contendo _endpoints_ na mesma zona. Isso irá permitir uma outra melhoria significativa de escalabilidade.
```
Comment (jhonmike) [inline content/pt/blog/_posts/2020-09-02-scaling-kubernetes-networking-endpointslices.md:42]:
```suggestion
O roteamento baseado por topologia irá atualizar o kube-proxy para dar preferência no roteamento de requisições para a mesma região ou zona, utilizando-se de campos de topologia armazenados em cada endpoint dentro de um EndpointSlice. Como uma melhoria futura disso, estamos explorando o potencial de subconjuntos de endpoint. Isso irá permitir o kube-proxy apenas observar um subconjunto de EndpointSlices. Por exemplo, isso pode ser combinado com o roteamento baseado em topologia e assim, o kube-proxy precisará observar apenas EndpointSlices contendo _endpoints_ na mesma zona. Isso irá permitir uma outra melhoria significativa de escalabilidade.
```

**Comment by rikatz** [comment]
@jhonmike thanks for the suggestion, commited them. Added the merge squash method so Prow will squash the commits when the PR is accepted :)

**Review by jhonmike — APPROVED** [review]
LGTM

**Comment by rikatz** [comment]
@jhonmike can you issue also a /lgtm in a comment, so this gets merged? 

Thanks!!

**Comment by jhonmike** [comment]
/lgtm

Sorry @rikatz, it was automatic

**Pull request merged**